reporting/reporting/Stored Procedures/DimMonthData.sql

```diff
@@ -0,0 +1,21 @@
+﻿
+
+CREATE PROCEDURE [dbo].[DimMonthData]
+
+AS
+BEGIN
+	
+	DECLARE @StartDate date= '20100101'
+
+	WHILE @StartDate <= '20151231'
+		BEGIN
+			INSERT INTO dbo.DimMonth
+			(
+				MonthId, MonthName, Year
+			)
+			VALUES ( CAST ( CAST( YEAR(@StartDate) AS VARCHAR(10) ) + CAST( MONTH(@StartDate) AS VARCHAR(10) ) AS INT), 
+			DATENAME( month , DateAdd( month , MONTH(@StartDate) , 0 ) - 1 ), YEAR(@StartDate) )
```

Comment:
@Sreenitha-Sampath DateAdd is a built-in function so has to be in uppercase just like you did in other places.